crates/bevy_math/src/curve/mod.rs

```diff
@@ -1096,6 +1096,10 @@ mod tests {
         });
     }
 
+    #[expect(
```

Comment:
can we put this on the assert! instead of the function so it's more clear which line is triggering clippy?

Reply:
that sounds like you put it inside the `assert!` instead of above.
```rust
#[expect(
            clippy::neg_multiply,
            reason = "Clippy doesn't like this, but it's correct"
        )]
        assert_eq!(
            mapped_curve.sample_unchecked(-1.0),
            (-1.0 * 3.0 + 1.0) / 7.0
        );
```
seems to work for me.

Reply:
nvm, it's giving an error about not applying to macros